test/shmemx/c11_test_shmemx_wait_until_vector.c

```diff
@@ -91,10 +91,10 @@
                                                                                                                      \
         int ncompleted = 0;                                                                                          \
         while(ncompleted < npes){                                                                                    \
-            int ndone = shmemx_wait_until_some_vector(ivars, npes, indices, status, SHMEM_CMP_EQ, cmp_values);       \
-            for(i = 0; i < ndone; i++){                                                                              \
-                total_sum += ivars[indices[i]];                                                                      \
-                status[indices[i]] = 1;                                                                              \
+            size_t ndone = shmemx_wait_until_some_vector(ivars, npes, indices, status, SHMEM_CMP_EQ, cmp_values);    \
+            for(size_t j = 0; j < ndone; j++){                                                                       \
+                total_sum += ivars[indices[j]];                                                                      \
+                status[indices[j]] = 1;                                                                              \
             }                                                                                                        \
```

Comment:
Not that it matters, but is there any reason of replacing `i` by `j`?

Reply:
Sorry, I just saw this question... yeah, we could/should have used `i`, but hopefully that's ok.